fog/view/server/test-utils/src/lib.rs

```diff
@@ -264,20 +266,14 @@ pub async fn assert_e_tx_out_records(
     }
 }
 
-/// Takes the TxOutSearchResults and interprets the ciphertext as follows:
-/// (1) The first byte is the delta between the max payload length and the
-/// length of this payload.
-/// (2) The rest of the bytes for that length are part of the ciphertext.
-fn interpret_tx_out_search_results(results: &[TxOutSearchResult]) -> HashSet<TxOutSearchResult> {
+fn interpret_tx_out_search_results(results: &[TxOutSearchResult]) -> Vec<TxOutSearchResult> {
```

Comment:
:question: Is this needed anymore?  I though the main point of this was to read the first byte and adjust the ciphertext

Reply:
Yes it's needed because the client will now use the `payload_length` field to interpret the `tx_out_search_result` `payload` rather than the first byte of the payload.